tests/checks/mock/test_disk.py

```diff
@@ -168,6 +168,20 @@ def test_no_psutil_freebsd(self, mock_df_output, mock_statvfs):
 
         self.coverage_report()
 
+    @mock.patch('utils.subprocess_output.get_subprocess_output',
+                return_value=(Fixtures.read_file('centos-df-Tk'), "", 0))
+    @mock.patch('os.statvfs', return_value=MockInodesMetrics())
+    def test_no_psutil_centos(self, mock_df_output, mock_statvfs):
+        self.run_check({'instances': [{'use_mount': 'no',
+                                       'excluded_filesystems': ['devfs'],
+                                       'excluded_disk_re': 'zroot/.+'}]},
```

Comment:
we could use instance options that make sense for this specific test for `excluded_filesystems` and `excluded_disk_re` (I could think of either having none of these options or the same options as the debian test)